libs/designsystem/header/src/header.component.ts

```diff
@@ -109,22 +116,19 @@ export class HeaderComponent implements AfterContentInit, OnInit {
       ],
       parent: this.injector,
     });
-
-    if (this.titleMaxLines > 0) {
-      this.fitHeadingConfig = {
-        maxLines: this.titleMaxLines,
-      };
-    }
   }
 
-  ngAfterContentInit(): void {
+  private onPotentialAvatarRendered(): void {
     // If an avatar is present we default to centered layout - unless configured otherwise:
-    if (this.avatar && this.centered === undefined) {
+    const avatarPresent: boolean = this.avatarContainerElement?.nativeElement.children.length > 0;
+    if (avatarPresent && this.centered === undefined) {
       this.centered = true;
     }
 
     if (this.centered) {
       this.actionGroupConfig.isCondensed = true;
     }
+
+    this.avatarReadyToRenderAfterContentProjection = true;
```

Comment:
Necessary due to the way kirby-progress-circle determines it's placement on the screen in relation to content projected content.

Reply:
Can no longer reproduce